tests/test_ssl.py

```diff
@@ -0,0 +1,25 @@
+# import asyncio
+# import ssl
+#
+# from aiomysql import create_pool
+#
+#
+# async def main():
+#     ctx = ssl.SSLContext(ssl.PROTOCOL_SSLv23)
+#     ctx.check_hostname = False
+#     ctx.verify_mode = ssl.CERT_NONE
+#
+#     # mysql_clear_password for connecting to rds
+#     async with create_pool(host='127.0.0.1', port=3308, user='root',
+#                            password='test1234',
```

Comment:
Normally you would call AWS's generate_db_auth_token() and use that as a password

```rds = boto3.client('rds')
      token = await asyncio.get_event_loop().run_in_executor(
                            None,
                            functools.partial(rds.generate_db_auth_token,
                                                         hostname, port, username, region))))```

Reply:
Yes, normally the client performs some form of hashing of the password before transmission, but for the MySQL IAM plugin to work it needs to see the token in cleartext, hence it requires TLS.

Also just making use of the free advertising checkout `aioboto3` :D

Reply:
Indeed. I wish I had found aioboto3 last December.